generators/app/templates/typescript/src/app/index.ts

```diff
@@ -0,0 +1,25 @@
+import * as Raven from "raven"; // Official `raven` module
+import * as RavenLambdaWrapper from "serverless-sentry-lib";
+import * as voxa from "voxa";
+import * as main from "./main";
+import * as alexaStates from "./states/alexa.states";
+import * as dialogflowStates from "./states/dialogflow.states";
+import * as variables from "./variables";
+import views from "./views";
+
+const app = new voxa.VoxaApp({ variables, views });
+
+main.register(app);
+
+// alexa
+export const alexaSkill = new voxa.AlexaPlatform(app);
```

Comment:
instead of `voxa.AlexaPlatform` better import voxa like `import { AlexaPlatform} from "voxa"` and then use `AlexaPlatform` directly